src/UserSpaceInstrumentation/MachineCode.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef USER_SPACE_INSTRUMENTATION_MACHINE_CODE_TOOLING_H
+#define USER_SPACE_INSTRUMENTATION_MACHINE_CODE_TOOLING_H
+
+#include <cstdint>
+#include <vector>
+
+namespace orbit_user_space_instrumentation {
+
+// Close to trivial tooling to build up a vector of bytes used to write machine code in a somewhat
+// structured way.
```

Comment:
Two suggestions for this sentence:
- Drop "Close to trivial" and "somewhat";
- The sentence is basically two sentences in one ("build up a vector of bytes" and "used to write machine code"), it's weird to read. I propose something along the lines of (but it can be improved) "Tooling to write machine code and build the corresponding vector of bytes in a structured way."

Reply:
done